projects/example.yaml

```diff
@@ -52,11 +52,226 @@ paths:
               x-modules:
                 - path: sys/page_save.js
             /parsoid:
+              x-modules:
```

Comment:
In light of #1180 you can keep this file intact.

Reply:
Reversed the changes.